src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

указывай названия свойств, которые анимируются
```suggestion
    transition: transform 300ms;
```

---

Исправил